packages/target-decisioning-engine/test/decisions.single.spec.js

```diff
@@ -86,6 +86,20 @@ describe("decisioning outcomes - single activity", () => {
                     content: {
                       doMagic: true,
                       importantValue: 150
+                    },
+                    responseTokens: {
+                      "activity.executionType": "client-side",
+                      "activity.id": 334411,
+                      "activity.name": "superfluous ab json",
+                      "activity.type": "ab",
+                      "experience.id": 0,
+                      "experience.name": "Experience A",
+                      "location.id": 0,
+                      "location.name": "superfluous-mbox",
+                      "location.type": "mbox",
+                      "offer.id": 631992,
```

Comment:
offer id is shown here, but not offer.name -- even though it is listed in the responseTokens array of the artifact 🤔

Reply:
The artifact shows that `offer.name` is indeed in `responseTokens`, but not in any `meta`.  So the SDK is doing the right thing.  But this could be a Smokescreen for not including the value in meta.